src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -144,15 +144,25 @@ const Cookies = ({ request }: { request: RequestSummary }) => {
   );
 };
 
-const StackTrace = ({ cx, frames }: { cx: any; frames: WiredFrame[] }) => {
+const StackTrace = ({
+  cx,
+  frames,
+  request,
+}: {
+  cx: any;
+  frames: WiredFrame[];
+  request: RequestSummary;
+}) => {
   const dispatch = useDispatch();
-  const selectFrame = (cx: any, frame: WiredFrame) => dispatch(actions.selectFrame(cx, frame));
-
+  const selectFrame = async (cx: any, frame: any) => {
+    console.log({ frame });
+    dispatch(seekToRequestFrame(request.id, frame.id, cx));
```

Comment:
Why not pass in the `RequestSummary` that we have instead of extracting its `id` and then fetching it from redux in the thunk?